pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -216,6 +218,14 @@ func NewPolicyApplier(rootNamespace string,
 	}
 }
 
+// Remove all Non-Alphanumeric Characters from a jwksURI and append to "n"
+func createFakeJwks(jwksURI string) string {
+	encodedString := b64.URLEncoding.EncodeToString([]byte(jwksURI))
+	reg := regexp.MustCompile("[^a-zA-Z0-9]+")
+	processedString := reg.ReplaceAllString(encodedString, "")
+	return fmt.Sprintf(`{"keys":[ {"e":"AQAB","kid":"abc","kty":"RSA","n":"failedToFetchJwksFor%s"}]}`, processedString)
```

Comment:
can we insert an extra `_`, i.e. `"failedToFetchJwksFor_%s"` to make it a bit more readable.